src/test/java/bio/terra/service/snapshotbuilder/query/filtervariable/SimpleFilterVariableForTests.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.service.snapshotbuilder.query.filtervariable;
+
+import bio.terra.service.snapshotbuilder.query.FilterVariable;
+import bio.terra.service.snapshotbuilder.query.TableVariable;
+import java.util.List;
+
+public class SimpleFilterVariableForTests implements FilterVariable {
```

Comment:
If this is for tests only it should be in the `test` source hierarchy

Reply:
I think it is?

Reply:
weird, I thought github was showing it outside `test`, not sure what I was seeing.